### ISSUE TITLE
Future Trends #7

### ISSUE THREAD

**Issue by mbedford001** [mbedford001/Bibliography#45]
### Title
The Aragon Research Globe™ for Conversational AI in
the Intelligent Contact Center (ICC), 2023
### URL
https://pro.cresta.com/hubfs/2023-02v%20The%20Aragon%20Research%20GlobeTM%20for%20Conversational%20AI%20in%20the%20ICC%2c%202023.pdf
### Key Points 
- The first wave of AI chatbots relied on scripted 
answers to resolve user questions since AI technology wasn’t mature enough at the time to enable chatbot conversations to be fluid and dynamic
-  there have been major improvements in AI software algorithms and the hardware that power them. The introduction of modern transformer neural network models that take into account all the words in a sentence at once provided significant improvements in accuracy
- Aragon predicts that these capabilities will continue to evolve and mature in the coming years (see Figure 1).
- On the hardware front, there have been major innovations in GPUs, FPGAs, and TPUs. GPUs and FPGAs were originally designed for other purposes but were found to be very good at processing AI workloads due to their ability to run many tasks in parallel.
- Each of these hardware solutions is very efficient in running parallel operations which are typical in a large language model.
- Modern AI chatbots have the ability to identify and keep track of multiple intents within a sentence and refer back to them throughout an entire disjointed, multi-threaded conversation. This capability has enabled chatbots to provide a more human-like contextual understanding, which allows for real intelligent conversations between people and machines. 
- Up until recently, the majority of chatbots have been text-based, with only a few leading providers embracing voice-based conversations; however, that is changing big time, with some of the leading providers today offering customizable voices for an IVA and in some cases, a customizable life-like avatar. 
- Natural language generation (NLG) has been lagging behind natural language understanding (NLU) for quite some time as the primary focus with conversational AI was on understanding intents from user interactions so correct actions could be identified. In the last several years, NLG has greatly improved, where now IVAs and chatbots are being deployed that can engage in intelligent and naturally flowing bi-directional conversations with users. 
- Even with all the recent advancements in AI and NLP, IVAs that are designed to focus on a specific use case are far more likely to be successful than those designed to serve a more general purpose. Natural language is fraught with many phrases and words with multiple meanings that attempting to understand intent in a very general way can be extremely complex and prone to errors. 
- many conversational AI providers are delivering solutions that are pre-trained out-of-the-box, typically targeted at a specific vertical solution or industry.
- 

### Citation
@misc{kennedy2023aragon,
  title={The Aragon Research Globe{\texttrademark} for Conversational AI in the Intelligent Contact Center (ICC), 2023},
  author={Kennedy, Craig},
  year={2023},
  month={January 17},
  publisher={Aragon Research},
  url={https://doi.org/Aragon Research Inc.}
}
